### PR TITLE
fixing miss-leading tested acc values

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

for some reason there were acc tested as loss for Boring models which does not make any sense and also leads to random failers...

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@rohitgr7 mind checks why these tests are failing, not sure how it happens that the predictions are negative...

**Comment by rohitgr7** [comment]
@Borda can we do it separately for accelerators and trainer since only those two modules are left? After that, we can remove the unused code (related to EvalModelTemplate) once everything is done. Will try to get some time by this weekend and complete them. Can we close this for now?

**Review by s-rog — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Comment by rohitgr7** [comment]
getting this error here:
```
RuntimeError: CUDA error: all CUDA-capable devices are busy or unavailable
```

aren't all the jobs running independently?? @Borda

**Comment by Borda** [comment]
> aren't all the jobs on azure running independently?

yes it should, each shall use own machine

**Comment by rohitgr7** [comment]
pls do a re-review. Changed/Updated a few things here.

File: tests/accelerators/test_ddp.py [hunk 1]
```diff
@@ -73,7 +66,7 @@ def test_multi_gpu_model_ddp_fit_test(tmpdir, cli_args):
 
     model_outs = result['result']
     for out in model_outs:
-        assert out['test_acc'] > 0.90
+        assert out['test_acc'] > 0.78
```
Comment (Borda) [inline tests/accelerators/test_ddp.py:69]:
```suggestion
        assert out['test_acc'] > 0.7
```

**Comment by Borda** [comment]
> pls do a re-review. Changed/Updated a few things here.

GREAT, Thank you!

**Pull request merged**